examples_tests/44.LevelCurveExtraction/main.cpp

```diff
@@ -363,13 +395,9 @@ int main()
         assert(false);
     float levelCurveSpacing = 10.0f;
     float planeX = 0, planeY = 1, planeZ = 0;
-    SLinesSettings lineSettings;
-    lineSettings.spacing = levelCurveSpacing;
-    lineSettings.x = planeX;
-    lineSettings.y = planeY;
-    lineSettings.z = planeZ;
-    auto linesBuffer = driver->createDeviceLocalGPUBufferOnDedMem(268435456);   //256 MB = 2^28B = 268435456 B
-    auto uniformLinesSettingsBuffer = driver->createFilledDeviceLocalGPUBufferOnDedMem(roundUp(4 * sizeof(float), 16ull), &lineSettings);
+    constexpr auto linesBufferSize = 268435456u; //256 MB = 2^28B = 268435456 B
```

Comment:
you need to declare the downstreaming buffer (in `SIrrlichtCreationParameters` at the start) to be of equal or larger size, otherwise download would need to happen in chunks.